public/scripts/extensions/attachments/manager.html

```diff
@@ -108,8 +108,9 @@ <h3 class="chatAttachmentsTitle margin0 title_restorable">
     </div>
 
     <div class="actionButtonTemplate">
-        <div class="actionButton list-group-item flex-container flexGap5" title="">
+        <div class="actionButton list-group-item flex-container flexGap5" style="align-items: center;" title="">
```

Comment:
I actually tried to use the STailwind class, however, it seems like `list-group-item` overrides it, causing the icons to not be in place. While I can add `!important`, to avoid messing up any of the other code, I just manually overrided it in the HTML.